packages/apps/demo/src/formatters/titleRow.formatter.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { context, TitleRow } from '@lightningjs/ui-components';
+import tileFormatter from './tile.formatter';
+
+window.CONTEXT = context;
```

Comment:
Why add this to the formatter instead of something like index.js or App.js?

Reply:
I believe I had this here for debugging at some point. I have removed it now. Great catch.